Source/Widgets/Viewer/Viewer.js

```diff
@@ -914,8 +914,8 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         /**
          * Gets or sets the target frame rate of the widget when <code>useDefaultRenderLoop</code>
          * is true. If undefined, the browser's {@link requestAnimationFrame} implementation
-         * determines the frame rate.  This value must be greater than 0 and a value higher than
-         * the underlying requestAnimationFrame implementatin will have no effect.
+         * determines the frame rate.  If defined, this value must be greater than 0.  A value higher
+         * than the underlying requestAnimationFrame implementation will have no affect.
```

Comment:
Yep that's what happened.  It was right in one copy and wrong in the other copy, and I had edited stuff in the other one, overwriting the one that was already correct.